packages/js/package.json

```diff
@@ -21,13 +21,16 @@
     "@silverhand/essentials": "^1.1.4",
     "jose": "^4.3.8",
     "js-base64": "^3.7.2",
+    "lodash": "^4.17.21",
```

Comment:
just [lodash.get](https://www.npmjs.com/package/lodash.get)?

Reply:
Get it! Thx.